database/src/schemas/index.ts

```diff
@@ -112,6 +148,31 @@ export const resolvers = {
         }
     },
     Mutation: {
+        createOAuthUserData: async (_: any, args: any, context: Context) => {
+            if (args.userId === undefined || args.userId === '') {
+                return 400
+            }
+            if (args.refreshToken === undefined || args.refreshToken === '') {
+                return 400
+            }
+            if (args.accessToken === undefined || args.accessToken === '') {
+                return 400
+            }
+            if (args.data === undefined || args.data === '') {
```

Comment:
i think data is optional

@Cavonstavant can you check that ?

Reply:
Yes it is